examples/eight_schools/main_svi.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+import sys
+
+import torch
+
+import pyro
+import pyro.distributions as dist
+from data import J, sigma_tensor, y_tensor
+from pyro.infer import SVI, Trace_ELBO
+from pyro.optim import Adam
+
+softplus = torch.nn.Softplus()
+data = torch.stack([y_tensor, sigma_tensor], dim=1)
+
+
+def model(data):
+    y = data[:, 0]
+    sigma = data[:, 1]
+
+    eta_dist = dist.Normal(torch.zeros(J), torch.ones(J))
+    mu_dist = dist.Normal(loc=torch.zeros(1), scale=10 * torch.ones(1))
+    tau_dist = dist.HalfCauchy(loc=torch.zeros(1), scale=25 * torch.ones(1))
```

Comment:
same comment as above

Reply:
I would also suggest moving these into the `sample` statements to make it consistent with the `mcmc` model.

Reply:
@neerajprad Could you clarify what you meant by moving these into the `sample` statements? Thanks!

Reply:
@irustandi 
```diff
- eta_dist = dist.Normal(...)
- eta = pyro.sample('eta', eta_dist)
+ pyro.sample('eta', dist.Normal(...)
```